integration_tests/test_gravity.py

```diff
@@ -303,7 +305,61 @@ def check():
     wait_for_fn("send-to-ethereum", check)
 
 
-def test_gov_token_mapping(gravity):
+def test_multiple_attestation_processing(gravity):
+    if not gravity.cronos.enable_auto_deployment:
+        geth = gravity.geth
+        cli = gravity.cronos.cosmos_cli()
+
+        # deploy test erc20 contract
+        erc20 = deploy_contract(
+            geth,
+            CONTRACTS["TestERC20A"],
+        )
+
+        balance = erc20.caller.balanceOf(ADDRS["validator"])
+        assert balance == 100000000000000000000000000
+
+        amount = 10
+        # Send some eth and erc20 to all accounts
+        print("fund all accounts")
+        for name in ACCOUNTS:
+            address = ACCOUNTS[name].address
+            send_transaction(
+                geth, {"to": address, "value": 10**17}, KEYS["validator"]
+            )
+            tx = erc20.functions.transfer(address, amount).buildTransaction(
+                {"from": ADDRS["validator"]}
+            )
+            tx_receipt = send_transaction(geth, tx, KEYS["validator"])
+            assert tx_receipt.status == 1, "should success"
+
+        print("generate multiple send to cosmos")
+        recipient = HexBytes(ADDRS["community"])
+
+        denom = f"gravity{erc20.address}"
+        previous = cli.balance(eth_to_bech32(recipient), denom=denom)
+
+        multiple_send_to_cosmos(
+            gravity.contract, erc20, recipient, amount, KEYS.values()
+        )
+
+        def check_gravity_balance():
+            """
+            check the all attestation are processed at once by comparing
+            with previous block balance
+            """
+            nonlocal previous
+            current = cli.balance(eth_to_bech32(recipient), denom=denom)
```

Comment:
Alternatively, you can pass the height parameter explicitly

Reply:
sure